plugins.d/cgroup-network-helper.sh

```diff
@@ -22,7 +22,10 @@
 
 # -----------------------------------------------------------------------------
 
-export PATH="${PATH}:/sbin:/usr/sbin:/usr/local/sbin"
+# the system path is cleared by cgroup-network
+export PATH="/bin:/usr/bin:/sbin:/usr/sbin:/usr/local/bin:/usr/local/sbin"
```

Comment:
The `#!/usr/bin/env bash` at the top of the file seems to be getting a default `PATH` from somewhere (or it wouldn't be able to find `bash`!) so maybe this line isn't needed after all?

Reply:
I think is the default shell (`/bin/sh`) used by `cgroup-network` that does that. Before that line above, the PATH is `/usr/local/bin:/usr/bin` (at least on my fedora 27 - I added `set >/tmp/log` in the script to find it out).